test/functional/tests/misc/test_random_cache_ops.py

```diff
@@ -0,0 +1,545 @@
+        [
+            core
+            for cache_core in active_cores
+            for core in used_core_devices
+            if core.path == cache_core.core_device.path
+        ]
+    )
+
+    used_core_devices[:] = [
+        core
+        for core in used_core_devices
+        if core.path not in [device.core_device.path for device in active_cores]
+    ]
+
+    if random.choice([True, False]):
+        cache_to_stop.stop()
+        for device in used_cache_devices:
+            if device.path == cache_to_stop.cache_device.path:
+                available_cache_devices.append(device)
+                used_cache_devices.remove(device)
+        return f"Stopped cache {cache_to_stop.cache_id} with flush"
+    else:
+        cache_to_stop.stop(no_data_flush=True)
+        for device in used_cache_devices:
+            if device.path == cache_to_stop.cache_device.path:
+                available_cache_devices.append(device)
+                used_cache_devices.remove(device)
+        return f"Stopped cache {cache_to_stop.cache_id} without flush"
+
+
+def set_param(
+    available_cache_devices: list,
+    used_cache_devices: list,
+    available_core_devices: list,
+    used_core_devices: list,
+) -> str | None:
+    cache_list = get_caches()
+
+    if not cache_list:
+        return None
+
+    cache_to_set_param = random.choice(cache_list)
+    set_param_list = [
+        cache_to_set_param.set_seq_cutoff_parameters,
+        cache_to_set_param.set_cleaning_policy,
+        cache_to_set_param.set_params_alru,
+        cache_to_set_param.set_params_acp,
+        cache_to_set_param.set_promotion_policy,
+        cache_to_set_param.set_params_nhit,
+    ]
+    param_to_set = random.choice(set_param_list)
+    match param_to_set:
+        case cache_to_set_param.set_seq_cutoff_parameters:
+            if random.choice([True, False]):
+                random_threshold = Size(random.randint(1, 4194181))
+                random_policy = random.choice(list(SeqCutOffPolicy))
+                random_promotion_count = random.randint(1, 65535)
+                random_seqcutoff_params = SeqCutOffParameters(
+                    threshold=random_threshold,
+                    policy=random_policy,
+                    promotion_count=random_promotion_count,
+                )
+                cache_to_set_param.set_seq_cutoff_parameters(random_seqcutoff_params)
+                return (
+                    f"Changed seq-cutoff params on cache {cache_to_set_param.cache_id} to:\n"
+                    f"Threshold: {str(random_threshold)}\n"
+                    f"Policy: {str(random_policy)}\n"
+                    f"Threshold: {str(random_threshold)}\n"
+                )
+            else:
+                core_list = cache_to_set_param.get_core_devices()
+                if not core_list:
+                    return None
+                random_core = random.choice(core_list)
+                random_threshold = Size(random.randint(1, 4194181))
```

Comment:
All value ranges for configuration parameters should be placed somewhere as constants, to be able to update it easily in case of any requirement changes. It can be done in a separate PR, I am leaving this comment just as the reminder.